tests/core/test_ModelSerialization.py

```diff
@@ -252,3 +253,25 @@ def test_id_ordering_fallback_exists(self, Teacher):
             "OVER (ORDER BY teacher.name, teacher.id)"
             in query._order_by[0].table.__str__()
         )
+
+    def test_bindparam(self, Student):
+        context1 = uuid4().hex
+        context2 = uuid4().hex
+        assert Student.serialize(
+            to_return=['contextual_id'],
+            filter_by={'id': self.student1.id},
+            params={'context': context1}
+        )[0]['contextual_id'] == Student.serialize(
+            to_return=['contextual_id'],
+            filter_by={'id': self.student1.id},
+            params={'context': context1}
+        )[0]['contextual_id']
```

Comment:
Removing `query = query.populate_existing()` makes this test fail as expected :)